sdk/python/foundation-models/system/inference/README.md

```diff
@@ -0,0 +1,2 @@
+# Foundation Model Inferencing
+For inferencing foundation models on Azure ML, the curated container __foundation-model-inference__ is used. The container leverages both DeepSpeed-FastGen and VLLM to give optimized inferencing results. For more information on this container and to use it with foundation models, see the [text-generation example](https://github.com/Azure/azureml-examples/blob/main/sdk/python/foundation-models/system/inference/text-generation/llama-safe-online-deployment.ipynb). Currently, text generation and chat completion tasks are supported with more on the way.
```

Comment:
Can you add brief description of vLLM and DeepSpeed-MII and links for them?
For DeepSpeed-MII, please mention that it uses DeepSpeed-Inference and latest DeepSpeed-FastGen (for llama2 model family)

https://github.com/vllm-project/vllm
https://github.com/microsoft/DeepSpeed-MII
https://github.com/microsoft/DeepSpeed/tree/master/blogs/deepspeed-fastgen

and instead of last  sentence, maybe it's better to have a separate section for the Supported tasks
Supported Tasks
- text-generation <link>
- more to be added
Also please remove chat completion from the list